.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Run '...'
+2. See error
+If possible, please provide the task package you are using.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Output**
+If applicable, add output from the command line.
+You can wrap the output in a code block (\`\`\`) to make it more readable.
+For example:
+\`\`\`
+$ sinol-make --version
+sinol-make 0.1.0
+\`\`\`
+converts to:
+```
+$ sinol-make --version
+sinol-make 0.1.0
+```
```

Comment:
Maybe we can make a code block that only needs its content to be replaced by the user? So instead of giving an example with an explanation, we could provide a ready code block with an information that its inside needs to be copy-pasted.